### PR TITLE
Add the request to error response for apple pay validate

### PR DESCRIPTION
**Pull request by elizalucas**
Additional PR for debugging the apple pay validate (to be removed once issue is resolved)

### CODE REVIEW

File: server-middleware/apiApplePay.ts [hunk 1]
```diff
@@ -37,28 +37,24 @@ app.post('/validate', (req, res) => {
...
+      })
       .then((a) => {
         // return the json received from Apple Pay server unmodified
         res.send(a.data)
       })
       .catch((a) => {
         res.send({
-          message: a.message,
+          errorMessage: a.message,
+          request: requestData,
```
Comment (jcantwell) [inline server-middleware/apiApplePay.ts:57]: does this change make sensitive data like keys/certificates available?

Comment (elizalucas) [reply server-middleware/apiApplePay.ts:57]: It only makes the merchant identifier, domain name, and display name available, all of which were previously hard coded into the file

File: server-middleware/apiApplePay.ts [hunk 2]
```diff
@@ -37,28 +37,24 @@ app.post('/validate', (req, res) => {
...
+        httpsAgent,
+      })
       .then((a) => {
         // return the json received from Apple Pay server unmodified
         res.send(a.data)
       })
       .catch((a) => {
         res.send({
-          message: a.message,
+          errorMessage: a.message,
```
Comment (antiv0) [inline server-middleware/apiApplePay.ts:56]: i d say send ALL (checksum of cert/key), applePayUrl, every possible detail
if this is not due to domain verification revoked then it is down to SOMETHING being off, so better to verify all

**Review by antiv0 — APPROVED** [review]
you can send back a bit by a bit, and come back to have merged, wait for deployment... repeat until all is verified

all immediately make a big return object and verify all fields (as this issue doesn't seem trivial, likely it will fail until you understand all)

**Review by antiv0 — DISMISSED** [review]

**Review by antiv0 — APPROVED** [review]

**Review by huaweigu — APPROVED** [review]

**Pull request merged**